tgui/packages/tgui/interfaces/LogViewer.tsx

```diff
@@ -130,17 +131,19 @@ export const LogViewer = (props: any, context: any) => {
           />
         </>
       }>
-      <Window.Content fitted>
-        <Flex direction="column" height="100%">
+      <Window.Content>
+        <Stack vertical fill>
           {displayFilterOptions && (
-            <FilterOptions
-              type={filterType}
-              text={filterText}
-              flags={filterFlags}
-              onTypeChange={(type) => updateFilterType(type)}
-              onTextChange={(text) => setFilterText(text)}
-              onFlagsChange={(flags) => setFilterFlags(flags)}
-            />
+            <Stack.Item>
+              <FilterOptions
+                type={filterType}
+                text={filterText}
+                flags={filterFlags}
+                onTypeChange={(type) => updateFilterType(type)}
+                onTextChange={(text) => setFilterText(text)}
+                onFlagsChange={(flags) => setFilterFlags(flags)}
+              />
+            </Stack.Item>
           )}
           <LogCategoryBar
```

Comment:
You need to wrap this in a Stack.Item with grow property (as I desribed it to you in Discord), and section must have a "fill" property. Stack -> Stack.Item hierarchy has to be maintained.